ext/opencensus_trace.c

```diff
@@ -64,6 +64,8 @@ void opencensus_trace_rinit()
 
     OPENCENSUS_G(current_span) = NULL;
     OPENCENSUS_G(trace_id) = NULL;
+    ALLOC_HASHTABLE(OPENCENSUS_G(baggage));
+    zend_hash_init(OPENCENSUS_G(baggage), 16, NULL, NULL, 0);
```

Comment:
We are not releasing the memory allocated for the baggage. We can release memory in trace shutdown hook 

eg:
```
void opencensus_trace_rshutdown()
{
    /* cleanup baggage */
    zend_hash_destroy(OPENCENSUS_G(baggage));
    FREE_HASHTABLE(OPENCENSUS_G(baggage));
}
```